tailwind.config.js

```diff
@@ -8,6 +8,25 @@ module.exports = {
   purge: [],
   theme: {
     extend: {
+      spacing: {
+        '14px': '14px',
+        '40px': '40px',
+        30: '7.5rem',
+      },
+      colors: {
```

Comment:
@CrazyOptimist below there is a color key which was wrongly named by me in the first commit I did and should be colors instead, like added here. So these colors (gray, blue…) and the ones below should be in the same colors object

Reply:
FYI, when I merge your branch into feature/nft-list I'll tackle this issue

Reply:
Thank you for the info, so you're going to move them to `colors`, okay.